src/database.py

```diff
@@ -188,6 +188,8 @@ class ChatRoom(Base):
     buyer_id = Column(UUID, ForeignKey("users.id"), nullable=False)
     is_deal_closed = Column(Boolean, nullable=False, server_default="f")
     is_revealed = Column(Boolean, nullable=False, server_default="f")
+    is_archived_buyer = Column(Boolean, nullable=False, server_default="f")
```

Comment:
If we do it like this, doesn't seem easy to get queries like "get all chat rooms in which the current user archives"